httprequest_lego_provider/tests/conftest.py

```diff
@@ -0,0 +1,59 @@
+# Copyright 2024 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+"""Fixtures for unit tests."""
+
+import base64
+import secrets
+
+import pytest
+from django.contrib.auth.models import User
+
+from httprequest_lego_provider.forms import FQDN_PREFIX
+from httprequest_lego_provider.models import Domain, DomainUserPermission
+
+
+@pytest.fixture(scope="module")
+def username():
+    """Provide a default username."""
+    yield "test_user"
```

Comment:
May I ask why `yield` is used here rather than `return`? Would this not create multiple contexts?

Reply:
AFAIK there isn't any difference in this case